### ISSUE TITLE
CtrlSpace is collapse with lines 

### ISSUE THREAD

**Issue by bearmingo** [vim-ctrlspace/vim-ctrlspace#34]
![selection_004](https://cloud.githubusercontent.com/assets/1488335/3219082/b034b33c-eff1-11e3-8a5b-9ea4fb25bf2b.jpg)

That make me so uncomfortable I do not know what's wrong in my vim.

**Comment by montefra** [comment]
I've just noticed the same problem when working on jinja2 templates with the [vim-jinja2](https://github.com/Glench/Vim-Jinja2-Syntax) plugin. Any time I open a html/jinjia file, the ctrlspace split get messed up. I don't recall the problem in any other case.

In case you need you can fine my [.vim here](https://github.com/montefra/.vim)

I'll test other two jinja plugins and let you know.

**Comment by montefra** [comment]
I've tested the two other plugins that I've found:

* https://github.com/lepture/vim-jinja gives me the same problem
* https://github.com/mitsuhiko/vim-jinja doesn't mess up ctrlspace window, but it's not usable (switching between files looses the jinja filetype...)

**Comment by Irubataru** [comment]
Hi, I know that this is pretty old, and that the project seems to have stagnated (which is really such a shame because this is the single most transformative vim-pugin I have used so far), but I encountered this bug and spent some time trying to track it down. I found a pair of settings that when combined together produced this error for me every time I opened a markdown document. I know, it is really weird.

I have a clean install with only your plugin and nothing else, and this in my vimrc file:

```
set nocompatible
filetype plugin on

call plug#begin("~/.vim/extensions")
Plug 'vim-ctrlspace/vim-ctrlspace'
call plug#end()

set hidden
set shiftwidth=2
set foldmethod=indent
```

So basically the latter combination causes this bug for me every time I open a markdown file for some reason, I have tried both with neovim v0.2.2 and with vim 8.0.1476, they both give the same error.
